native/README.md

```diff
@@ -0,0 +1,16 @@
+# Notes for Android
+
+## Contributing
+
+* Use node 8 and run `npm install`
+* Plugin an Android phone and make sure it is visible via `adb devices`
+* Run `npm start`
+
+## Useful commands
+
+`npm run dm` - when the app is open this opens the debug menu'
+
+Use `npx` (from npm 5) to run commands for `react-native`
```

Comment:
Since `react-native` is already a dependency, why not just make the two scripts below npm scripts so you don't need to mess w/ `npx` and [re]downloading packages on-the-fly (or mess with using `$(npm bin)/react-native upgrade` or `./node_modules/.bin/react-native upgrade`)?